#!/usr/bin/env python

PACKAGE = 'jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *;

gen = ParameterGenerator ()

gen.add("kernel_size", int_t, 0, "Size of the kernel (only one because we use a square window). Must be odd.", 7, 0, 50)

exit (gen.generate (PACKAGE, "jsk_perception", "MedianFilter"))
